src/models/order.ts

```diff
@@ -0,0 +1,52 @@
+import { Model, Sequelize, DataTypes } from "sequelize";
+
+export class Order extends Model {
+  public id!: string;
+  public productId!: string;
+  public userId!: string;
+  public quantity!: number;
+  public price!: number;
+  public status!: string;
+  public readonly createdAt!: Date;
+  public readonly updatedAt!: Date;
+}
+
+export const orderMap = (sequelize: Sequelize): void => {
+  Order.init(
+    {
+      id: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        primaryKey: true,
+      },
+      productId: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      userId: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      quantity: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+      },
+      price: {
+        type: DataTypes.FLOAT,
+        allowNull: false,
+      },
+      status: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+    },
+    {
+      sequelize,
+      modelName: "Order",
+      tableName: "orders",
```

Comment:
still the same??

Reply:
noo I changed, maybe in the same time like you checked:)